app/jobs/scheduled/dashboard_stats.rb

```diff
@@ -1,16 +1,7 @@
 module Jobs
   class DashboardStats < Jobs::Scheduled
-    every 30.minutes
-
-    def execute(args)
-      stats = AdminDashboardData.fetch_stats.as_json
-
-      # Add some extra time to the expiry so that the next job run has plenty of time to
-      # finish before previous cached value expires.
-      $redis.setex AdminDashboardData.stats_cache_key, (AdminDashboardData.recalculate_interval + 5).minutes, stats.to_json
-
-      stats
-    end
+    include Jobs::Stats
```

Comment:
Opps I see what you mean. I didn't realize that the job wouldn't receive the right argument when ran in the background.  I'll see if there is a way to pass it any argument. Otherwise, I'll extract the logic for `$redis.setex` into a different method which can be reused.